Add case study: CW20 (#44)
softwaresaved/event-organisation-guide: docs/eog/case-studies-cw20-agenda.rst

.. _CW20-Agenda: 

CW20 Agenda
============

For in-person CWs, the days are long to maximise the opportunities for networking, collaboration, discussions, and solving problems across the research software landscape. 
The agenda that we had spent months organising was packed full of talks, group sessions, social activities and mini-workshops - we originally had 18 mini-workshops scheduled to take place across five sessions.

> If we had organised CW20 as an online event from the beginning, there are many things we would have done differently - but since we found ourselves in the position of having to transition from an in-person event to an online one, we still wanted to incorporate as much of the original programme as possible. 

Reviewer: ```suggestion
If we had organised CW20 as an online event from the beginning, there are many things we would have done differently - but since we found ourselves in the position of having to transition from an in-person event to an online one at short notice, we still wanted to incorporate as much of the original programme as possible. 
```

We contacted speakers and workshop facilitators to ask if they were still able to deliver in the online setting. 
In the end, two of the workshops dropped out and we were therefore able to shift the others around so that we had 16 workshops across four sessions (with four workshops in parallel for each session). 

> Because we were now unable to carry out the planned social programme (activities such visiting the Ulster Museum, a walk around the Belfast Botanic Gardens and of course the conference dinner), we were able to shorten the days and add in further breaks and changeover/buffer time. 

Reviewer: ```suggestion
Because we were now unable to carry out the planned social programme (activities such visiting the Ulster Museum, a walk around the Belfast Botanic gardens and of course the conference dinner), we were able to shorten the days and add in further breaks and changeover/buffer time. 
```

Breaks were also scheduled in such a way to allow time for the creation of breakout rooms for the next session.

Sessions
---------

Icebreakers (Networking)
^^^^^^^^^^^^^^^^^^^^^^^^^

We started each day with an icebreaker to get participants used to the infrastructure and to have a chance to chat in small groups. 
We asked a couple of yes or no questions for participants to respond to using `Zoom’s non-verbal feedback feature <https://support.zoom.us/hc/en-us/articles/115001286183-Nonverbal-Feedback-During-Meetings>`_, and then randomly assigned them into breakout rooms of 3-4 people to discuss the following prompts which we thought would be both fun and useful:

- What are you reading, watching and/or listening to at the moment (and why)?
- What advice/resources/tools/guides do you find useful for working remotely (and why)?

We then asked participants to write down their answers in the note-taking document during a few minutes of “silent documenting”, which had the added benefit of empowering people to directly contribute to the shared notes (`example icebreaker document <https://doi.org/10.6084/m9.figshare.12498275>`_).

Keynote Talks (Informing)
^^^^^^^^^^^^^^^^^^^^^^^^^^

> Keynote speakers were contacted prior to the event to confirm the time of talk and format, and no time keeping was necessary on the day. 

Reviewer: ```suggestion
Keynote speakers were contacted prior to the event to confirm the time of their talk and its format, and enable no additional time keeper being needed for their session on the day. 
```

The Chair of the session introduced the speaker and reminded the participants that they could take notes and ask questions within the session’s section of the day’s collaborative notes where a link to the slides was also shared. 
The speaker shared their presentation slides via `Screen Share in Zoom <https://support.zoom.us/hc/en-us/articles/201362153-Sharing-your-screen>`_, and the Host set `Spotlight Video <https://support.zoom.us/hc/en-us/articles/201362653-Spotlight-Video>`_ for the speaker so that they would be visible to all participants and in the recording. 

> During the Q&A, the Chair then relayed some of the questions from the notes to the speaker, and the remainder of the questions were answered in-line within the note-taking document.

Reviewer: ```suggestion
During the Q&A, the Chair then relayed some of the questions from the notes to the speaker, and the remainder of the questions were answered in-line within the note-taking document after the session.
```

You can read more details about how we ran the Collaborative Ideas session in the Collaborations Workshop section of the `CSCCE's guide to Using virtual events to facilitate community building: event formats <https://doi.org/10.5281/zenodo.3934384>`_.

Mini-workshops and Demos (Teaching)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

We had 16 `Mini-workshops and Demos <https://www.software.ac.uk/cw20/mini-workshops-and-demo-sessions>`_ across four sessions. 
We used a separate Zoom account for each of the four parallel workshops in each session, with an assigned Host (and Co-host for backup) for handling the logistics of the workshop (managing Zoom, recording the session, pointing to the relevant notes document) and helping the workshop facilitator with any technical issues. 
We pointed people to the different Zoom room links in the day’s collaborative notes and in the Slack channel, and someone stayed in the main room to help people get to where they wanted to go. 
A lesson learned is to provide guidance to workshop facilitators that the workshops should include participant interaction for at least half of the time, and not just one long presentation. 

> You can find the full collection of `CW20 infrastructure and documentation resources on Figshare <https://doi.org/10.6084/m9.figshare.c.5026400>`_.

Reviewer: This sentence seems to float on it's own - it's not part of the Mini-workshops and Demos (Teaching) section - is it needed here? - if so perhaps give it it's own heading e.g. 'Infrastructure to help run the session' or put it at the end of the introductory paragraph on the page
